content/tutorials/r_setup_codingclub_session/index.Rmd

```diff
@@ -17,12 +17,12 @@ knitr::opts_chunk$set(eval = FALSE)
 ```
 
 
-Before each coding club you will usually have to download some data and some scripts.
-This can be done manually by going to each of the GitHub pages but it is much easier to use the `setup_codingclub_session()` function of the `inborutils` package.
+Before each coding club you will usually have to download some datasets and some scripts.
+This can be done manually from the correspondent GitHub webpages but it is much easier to use the `setup_codingclub_session()` function of the `inborutils` package.
 
-To begin make sure that you have the latest version of `inborutils` installed by running the following code:
+To begin, make sure that you have the latest version of `inborutils` installed by running the following code:
 
-```{r}
+```{r install_inborutils_if_not_present}
```

Comment:
it's generally not a good idea to use underscores in R chunk names. R markdown cannot handle them well if you want to refer them. Best use hyphens "-". Applies to this and other chunks.

Reply:
ah, nice to know! I will!

Reply:
https://github.com/inbo/tutorials/pull/206/commits/b295297ab6f2d0428eb8994f24f222e1a4e35873